core.js

```diff
@@ -1602,7 +1662,7 @@ class FileTypeParser {
 	}
 }
 
-export async function fileTypeStream(readableStream, {sampleSize = minimumBytes} = {}) {
+export async function fileTypeStream(readableStream, {sampleSize = minimumBytes} = {}, fileTypeOptions) {
```

Comment:
Inconsistent with TypeScript declaration. 

Why is the class constructor not used to initialize with the `fileTypeOptions`?

Reply:
Well, from the code structure, this method seemed a bit distinct (being separated from the others). I also found it a bit more intuitive that the class constructor is not used here, to better indicate that the stream is not stored.

I don't have a strong inclination here, so we could also use the class constructor. In case you don't agree to my reasoning?

Reply:
I agree a it is a bit different indeed, but don't see a good reason to treat it any different.

Reply:
Adjusted.